doc/contributing/releases.md

```diff
@@ -944,6 +944,9 @@ a `NODEJS_RELEASE_HOST` environment variable:
 NODEJS_RELEASE_HOST=proxy.xyz ./tools/release.sh
 ```
 
+> In case <https://nodejs.org/dist/index.json> or <https://nodejs.org/dist/index.tab> are not
+> updated or the `SHASUMS256.txt` has not been generated correctly, you can repeat the step.
```

Comment:
```suggestion
> In case <https://nodejs.org/dist/index.json>, <https://nodejs.org/dist/index.tab>,
> or`SHASUMS256.txt` have not been generated correctly, you can repeat the step.
```